### PR TITLE
WIP: Add `dist` command

### PR DESCRIPTION
**Pull request by danielpgross**

### CODE REVIEW

File: dev/lib/product_taxonomy/commands/command.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require "logger"
+require "benchmark"
+
+module ProductTaxonomy
+  class Command
+    attr_reader :logger, :options
+
+    def initialize(options)
```
[the remark this answers is not in the document]
Comment (elsom25) [reply dev/lib/product_taxonomy/commands/command.rb:10]: I also wonder if we should inject the logger itself candidly 🤔

File: dev/lib/product_taxonomy/models/attribute.rb [hunk 2]
```diff
@@ -29,6 +29,25 @@ def load_from_source(source_data)
...
+          "attributes" => all.map { _1.to_json(locale:) },
+        }
+      end
+
+      def to_txt(version:, locale: "en")
+        header = <<~HEADER
+          # Shopify Product Taxonomy - Attributes: #{version}
+          # Format: {GID} : {Attribute name}
+        HEADER
+        padding = 10
```
Comment (elsom25) [inline dev/lib/product_taxonomy/models/attribute.rb:44]: Have to be clever with padding size, but can come in followup PRs

File: dev/lib/product_taxonomy/models/value.rb [hunk 3]
```diff
@@ -28,6 +28,25 @@ def load_from_source(source_data)
...
+          "values" => all.map { _1.to_json(locale:) },
+        }
+      end
+
+      def to_txt(version:, locale: "en")
+        header = <<~HEADER
+          # Shopify Product Taxonomy - Attribute Values: #{version}
+          # Format: {GID} : {Value name} [{Attribute name}]
+        HEADER
+        padding = 10
```
Comment (elsom25) [inline dev/lib/product_taxonomy/models/value.rb:44]: Same padding comment here

**Review by elsom25 — APPROVED** [review]

**Comment by danielpgross** [comment]
Replaced by stack in #469

**Pull request closed without merging**